README.md

```diff
@@ -186,8 +186,8 @@ curl -X PUT -i -u myname:mypassword \
     http://localhost:8080/configuration
 ``` 
 
-where you should replace `myname` and `mypassword` with the credentials expected by WebLogic Server for its REST API,
-and `<path to yaml>` is the relative path to the configuration to use.
+Replace `myname` and `mypassword` with the credentials expected by WebLogic Server for its REST API,
+and `<path to yaml>` with the relative path to the configuration to use.
```

Comment:
Still missing some from the first time around:
Once the exporter is configured -> After the exporter is configured
Building a docker image -> Building a Docker image